consensus/istanbul/backend/backend.go

```diff
@@ -468,15 +471,24 @@ func (sb *Backend) Verify(proposal istanbul.Proposal) (time.Duration, error) {
 	if txnHash != block.Header().TxHash {
 		return 0, errMismatchTxhashes
 	}
-
-	// The author should be the first person to propose the block to ensure that randomness matches up.
-	addr, err := sb.Author(block.Header())
+	minVersion := &params.VersionInfo{Major: 1, Minor: 1, Patch: 0}
+	version, err := blockchain_parameters.GetMinimumVersion(nil, nil)
 	if err != nil {
-		sb.logger.Error("Could not recover orignal author of the block to verify the randomness", "err", err, "func", "Verify")
-		return 0, errInvalidProposal
-	} else if addr != block.Header().Coinbase {
-		sb.logger.Error("Original author of the block does not match the coinbase", "addr", addr, "coinbase", block.Header().Coinbase, "func", "Verify")
-		return 0, errInvalidCoinbase
+		if err != comm_errors.ErrRegistryContractNotDeployed && err != comm_errors.ErrSmartContractNotDeployed {
+			log.Debug("Error checking client version", "err", err, "contract", hexutil.Encode(params.BlockchainParametersRegistryId[:]))
+			return 0, err
+		}
+	} else {
+		if version.Cmp(minVersion) <= 0 {
```

Comment:
Do we expect this PR to be included in 1.1.0? If so, if the minimum client version was 1.1.0, wouldn't we want to allow the author to differ from the coinbase?

Reply:
My understanding was no we wouldn't want that: we want version 1.1 to enforce using/reading from the `header.Coinbase` but not writing to it.
Then version 1.1+ will allow them to change it.

Is this not what we discussed?

Reply:
Resolved via slack, we indeed want to use < 1.1 as asa described